src/AppInstallerCLITests/Experiment.cpp

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "TestCommon.h"
+#include "TestSettings.h"
+#include <winget/Experiment.h>
+#include <AppInstallerStrings.h>
+
+using namespace TestCommon;
+using namespace AppInstaller::Settings;
+using namespace AppInstaller::Experiment;
+
+#define ASSERT_EXPERIMENT(_isEnabled_, _toggleSource_) \
+    auto testExperimentState = Experiment::GetState(ExperimentKey::TestExperiment); \
+    REQUIRE(_isEnabled_ == testExperimentState.IsEnabled()); \
+    REQUIRE(_toggleSource_ == testExperimentState.ToggleSource());
+
+TEST_CASE("Experiment_GroupPolicyControl", "[experiment]")
+{
+    SECTION("Not configured")
+    {
+        Experiment::ResetStates();

```

Comment:
I would prefer this to be a struct that resets the state on destruction, like `GroupPolicyTestOverride` and `TestUserSettings`. Just to be sure that nothing we set here can escape and affect other tests